app/templates/_package.json

```diff
@@ -32,6 +32,7 @@
     "grunt-legacssy": "~0.4.0",
     "grunt-string-replace": "~1.0.0",
     "grunt-topdoc": "~0.3.0",
+    "jit-grunt": "~0.9.1",
     "load-grunt-tasks": "~1.0.0",
```

Comment:
Probbbbably. I feel like there was a cf-component where we needed both. Let me see if I can dig that up.